datanode/client/query_client.go

```diff
@@ -87,7 +88,12 @@ func (dc *dataNodeQueryClientImpl) QueryRaw(ctx context.Context, host topology.H
 }
 
 func (dc *dataNodeQueryClientImpl) queryRaw(ctx context.Context, host topology.Host, query queryCom.AQLQuery, hll bool) (bs []byte, err error) {
-	url := fmt.Sprintf("http://%s/query/aql?dataonly=1", host.Address())
+	// TODO: @shz remove this hack when we get proper address from host.Address()
+	address := host.Address()
+	if strings.HasPrefix(address, "http://") {
```

Comment:
better if you see any http:// https://, etc, then no need to prefix protocol
and how about sql query

Reply:
updated.

sql query will use the same client